hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneKey.java

```diff
@@ -45,6 +45,10 @@ public class OzoneKey {
    * Name of the Key.
    */
   private final String name;
+  /**
+   * Name of the Key owner.
+   */
+  private final String ownerName;
```

Comment:
In `OzoneVolume` and `OzoneBucket` this field is called `owner`. We should probably use the same name throughout this change.

Reply:
I will change this, but my feedback may be slow, since I am still in holiday until 30th Jan.